bin/tests/it/cmd_line_env.rs

```diff
@@ -104,18 +104,17 @@ fn test_list_config_from_conf() -> io::Result<()> {
         .arg("-l")
         .unwrap();
     assert!(output.status.success());
-    let stdout = from_utf8(&output.stdout).unwrap();
+    let output = from_utf8(&output.stderr).unwrap();
```

Comment:
tracing framework is using stderr for log output.  
https://github.com/logdna/logdna-agent-v2/blob/9b07f6782c96f1768acebaab85e7d7e4a8ee860c/bin/src/main.rs#L38